itests/pom.xml

```diff
@@ -248,6 +248,17 @@
         <version>${project.version}</version>
         <classifier>tests</classifier>
       </dependency>
+      <dependency>
```

Comment:
Wouldn't this dependency fail if the `hive-iceberg-handler` never compiled before on this machine?

Reply:
I added the `iceberg` profile to the `.travis.yml`

Reply:
The Travis will be removed soon. That should not be a problem. My fear is that when the developers starts to run the the tests on their local machines without the `iceberg` flag, they will have a problem (maybe only if they are offline)